Foundation/NSData.swift

```diff
@@ -383,8 +383,8 @@ extension NSData {
         }
         
         
-        return other.withUnsafeBytes { (bytes2: UnsafePointer<Void>) -> Bool in
-            let bytes1 = bytes
```

Comment:
Though UnsafePointer<Void> needs to be replaced by UnsafeRawPointer, I don't think its applicable to ```withUnsafeBytes<ResultType, ContentType>(_ body: (UnsafePointer<ContentType>) throws -> ResultType)```